src/components/data_display/helpers.ts

```diff
@@ -203,7 +215,12 @@ const buildColumnList = (
       ? 'tableViewExcludeColumns'
       : 'filterExcludeColumns'
 
-  const includeColumns = getIncludeColumns(includeField, dataView, allColumns)
+  const includeColumns = getIncludeColumns(
+    includeField,
+    dataView,
+    // Don't include "FilterData" columns in full list
+    allColumns.filter((col) => !new RegExp(FILTER_COLUMN_SUFFIX).test(col))
+  )
```

Comment:
The opposite of above -- *don't* include "filterData" columns in **table** view column list by default (they can still be explicitly included, though, if this is desired)